.github/workflows/JetNews.yaml

```diff
@@ -41,9 +41,9 @@ jobs:
             ~/.gradle/caches/build-cache-*
           key: gradle-${{ hashFiles('checksum.txt') }}
 
-      - name: Build project
+      - name: Build project and run local tests
         working-directory: ${{ env.SAMPLE_PATH }}
-        run: ./gradlew spotlessCheck assembleDebug lintDebug --stacktrace
+        run: ./gradlew spotlessCheck assembleDebug lintDebug testDebug --stacktrace
```

Comment:
This could just be `./gradlew build`?

Reply:
build runs all the release stuff, however buildDebug does not run local tests.